packages/cashier/src/Components/Error/__tests__/form-error.spec.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import FormError from '../form-error';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { routes } from '@deriv/shared';
+
+jest.mock('Stores/connect.js', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+describe('<FormError />', () => {
+    const portalRoot = global.document.createElement('div');
+    portalRoot.setAttribute('id', 'modal_root');
+    global.document.body.appendChild(portalRoot);
```

Comment:
since these are called in all tests, can we move these in `beforeAll` method? please also do a cleanup after by removing the element